test/WebSites/HtmlGenerationWebSite/Views/HtmlGeneration_Home/DuplicateAntiforgeryTokenRegistration.cshtml

```diff
@@ -0,0 +1,19 @@
+﻿@{
```

Comment:
> just multiple <form>s in a single view.

I believe we already have tests for this scenario. This is specifically for scenario where user is trying to call `@Html.AntiForgeryToken()` even though the beginform/beginrouteform already do that. I did not want to create 2 separate files for this negative scenario and hence clubbed them into one.